CoEpi/api/model/request/ApiParamsCenReport.swift

```diff
@@ -9,7 +9,7 @@ struct ApiParamsCenReport: Encodable {
 
 extension ApiParamsCenReport {
     init(report: MyCenReport) {
-        self.init(reportID: report.id, report: report.report.report, cenKeys: report.keys.joined(separator: ","),
+        self.init(reportID: report.report.id, report: report.report.report.toBase64() ?? "Decoding error".toBase64()! , cenKeys: report.keys.joined(separator: ","),
```

Comment:
Let's examine why `toBase64()` returns an optional. Maybe the utf-8 decoding can be force unwrapped, see https://stackoverflow.com/a/46152738/930450

Otherwise we should look for a different way to handle this.